docs/advanced-usage.rst

```diff
@@ -144,66 +183,146 @@ index ``cid``.
-        asset=tx_retrieved['transaction']['asset'],
-        signing_key=alice.signing_key,
+    >>> bob = generate_keypair()
+    >>> bob.verifying_key
+    'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA'
+
+And prepare the transfer transaction:
+
+.. code-block:: python
+
+    tx_transfer = bdb.transactions.prepare(
+        operation='TRANSFER',
+        inputs=input_,
+        asset=asset,
+        owners_after=bob.verifying_key,
     )
 
 The ``transfer_tx`` dictionary should look something like:
 
 .. code-block:: python
 
-    {'id': '0c037170ad062b963ece60925fc65518a8ff8bd7d84ff801dac8f7038f159d4f',
-     'transaction': {'asset': {'id': '8409a700-0929-4143-ae3c-f9b7d6c3c2a6'},
+    {'id': '7fde91ebdb05bb49af4e7613647c27cf4e30809efcbae28e6b34e138e0360c76',
+     'transaction': {'asset': {'id': 'd04b05de-774c-4f81-9e54-6c19ed3cd18d'},
       'conditions': [{'amount': 1,
         'cid': 0,
         'condition': {'details': {'bitmask': 32,
-          'public_key': 'EKWLdArKJ8C2vRjTFPTbYGkP8QZCuV3xwRmENm97Rief',
+          'public_key': 'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA',
           'signature': None,
           'type': 'fulfillment',
           'type_id': 4},
-         'uri': 'cc:4:20:xeWOvPSlHGEOKt-eEe3UBE3oa-L7Lfffo1fLIsdkhuw:96'},
-        'owners_after': ['EKWLdArKJ8C2vRjTFPTbYGkP8QZCuV3xwRmENm97Rief']}],
+         'uri': 'cc:4:20:yjsOmwsugrgj_QAcdaLZdZWKHWTB2T5yVmBf8IfdV_s:96'},
+        'owners_after': ['EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA']}],
       'fulfillments': [{'fid': 0,
-        'fulfillment': 'cf:4:0R5T1sXt94bEQbhZhd3ejwLhQxkY_iorNvwkmqkebXsPMWcOUwD-hKTIIRm8jIlcxDYD42clfPpJXpP9lu1xImd1ZFRt6xtAzSLGL-4k4Gi3-zP1JEIW_oOHecG0rjQK',
+        'fulfillment': {'bitmask': 32,
+         'public_key': '3Cxh1eKZk3Wp9KGBWFS7iVde465UvqUKnEqTg2MW4wNf',
+         'signature': None,
+         'type': 'fulfillment',
+         'type_id': 4},
         'input': {'cid': 0,
-         'txid': 'c3bc2f33aeaea36f1153c8d5604ba69f9289670ef1d7fcded80d3abf164a4a39'},
-        'owners_before': ['F5K2n1xnqoyCBiX2PaqjRrLiF6KsBaFzyVLE9WrySFKg']}],
+         'txid': '57cff2b9490468bdb6d4767a1b07905fdbe18d638d9c7783f639b4b2bc165c39'},
+        'owners_before': ['3Cxh1eKZk3Wp9KGBWFS7iVde465UvqUKnEqTg2MW4wNf']}],
       'metadata': None,
       'operation': 'TRANSFER',
-      'timestamp': '1477661596'},
+      'timestamp': '1479401063'},
      'version': 1}
 
 Notice, ``bob``'s verifying key (public key), appearing in the above ``dict``.
 
 .. code-block:: python
 
     >>> bob.verifying_key
-    'EKWLdArKJ8C2vRjTFPTbYGkP8QZCuV3xwRmENm97Rief'
+    'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA'
+
+The transaction now needs to be fulfilled by ``alice``:
+
+.. code-block:: python
+
+    signed_tx_transfer = bdb.transactions.fulfill(
+        tx_transfer,
+        private_keys=alice.signing_key,
+    )
+
+If you look at the content of ``signed_tx_transfer`` you should see the added
+fulfilment uri, holding the signature:
+
+.. code-block:: python
```

Comment:
I would just put the `signed_tx_transfer['transaction']['fulfillments'][0]['fulfillment']` block here.